### ISSUE TITLE
Do lit review and update examples in 20.5, non-XML-based approaches

### ISSUE THREAD

**Issue by JanelleJenstad** [TEIC/TEI#2438]
See [https://tei-c.org/release/doc/tei-p5-doc/en/html/NH.html#NHNX](https://tei-c.org/release/doc/tei-p5-doc/en/html/NH.html#NHNX)

**Comment by ebeshero** [comment]
Hi, @JanelleJenstad Think you can get us a solid start on this?

**Comment by ebeshero** [comment]
(And by solid start, maybe just flesh out the explanation of what you intended here...it's not super clear on its own.)

**Comment by JanelleJenstad** [comment]
@martindholmes directed my attention to this section of the guidelines and I noticed that the most recent reference is from 2007. If the Guidelines genuinely want to direct people to useful sources, I thought we should do a literature review and update the sources listed here. I'll do a quick search.

**Comment by JanelleJenstad** [comment]
Some sources from Balisage proceedings:

Pondorf, Denis, and Andreas Witt. “Freestyle Markup Language: Specification of an intuitive, powerful, polyhierarchical new extensible markup language.” Presented at Balisage: The Markup Conference 2010, Montréal, Canada, August 3 - 6, 2010. In Proceedings of Balisage: The Markup Conference 2010. Balisage Series on Markup Technologies, vol. 5 (2010). https://doi.org/10.4242/BalisageVol5.Pondorf01.

Piez, Wendell. “Hierarchies within range space: From LMNL to OHCO.” Presented at Balisage: The Markup Conference 2014, Washington, DC, August 5 - 8, 2014. In Proceedings of Balisage: The Markup Conference 2014. Balisage Series on Markup Technologies, vol. 13 (2014). https://doi.org/10.4242/BalisageVol13.Piez01.

Di Iorio, Angelo, Silvio Peroni and Fabio Vitali. “Towards markup support for full GODDAGs and beyond: the EARMARK approach.” Presented at Balisage: The Markup Conference 2009, Montréal, Canada, August 11 - 14, 2009. In Proceedings of Balisage: The Markup Conference 2009. Balisage Series on Markup Technologies, vol. 3 (2009). https://doi.org/10.4242/BalisageVol3.Peroni01.

Sperberg-McQueen, C. M. “Representing concurrent document structures using Trojan Horse markup.” Presented at Balisage: The Markup Conference 2018, Washington, DC, July 31 - August 3, 2018. In Proceedings of Balisage: The Markup Conference 2018. Balisage Series on Markup Technologies, vol. 21 (2018). https://doi.org/10.4242/BalisageVol21.Sperberg-McQueen01.

**Comment by ebeshero** [comment]
Yes—there’s also the Text as Graph lit more recently that we can add, by @bleekere and Ronald Dekker.

**Comment by ebeshero** [comment]
Council F2F Paderborn subgroup: We think reviewing and revising this chapter should be the work of a dedicated task force.
